src/pages/reference/cli/stacks.mdx

```diff
@@ -18,25 +18,69 @@ nitric stack new
 
 Follow the prompts to create a stack for your provider.
 
+<CodeGroup>
+
+```{{ title: "AWS" }}
+What should we name this stack? aws-stack
+Which provider do you want to deploy with? aws
+Which region should the stack deploy to? us-east-1
+```
+
+```{{ title: "Azure" }}
+What should we name this stack? azure-stack
+Which provider do you want to deploy with? azure
+Which region should the stack deploy to? eastus
+Provide the organisation to associate with the API: my-org
+Provide the Admin email to associate with the API: admin@company.co
 ```
-? What do you want to call your new stack? aws-stack
-? Which Cloud do you wish to deploy to? aws
-? select the region us-east-1
+
+```{{ title: "Google Cloud" }}
+What should we name this stack? gcp-stack
+Which provider do you want to deploy with? gcp
+Which region should the stack deploy to? us-east1
+Provide the gcp project ID to deploy to: project-id
 ```
 
+</CodeGroup>
+
 The stack definition will be created in the root of your project as a YAML file prefixed with `nitric-` e.g. `nitric-aws-stack.yaml`
 
-```yaml
+<CodeGroup>
+
+```yaml{{ title: "AWS" }}
 name: my-aws-stack
-provider: nitric/aws@0.24.0
+# Using @latest in production can lead to breaking changes. Pin to a specific version for stability.
+provider: nitric/aws@latest
```

Comment:
Are we keeping this as latest or doing `x.x.x` referring to Tim's comment earlier.

Reply:
keeping as latest for now